crates/fuel-core/src/state/historical_rocksdb.rs

```diff
@@ -265,15 +269,21 @@ where
             .next()
             .ok_or(DatabaseError::ReachedEndOfHistory)??;
 
+        self.rollback_block_at(latest_height)?;
+
+        Ok(latest_height)
+    }
+
+    fn rollback_block_at(&self, height_to_rollback: u64) -> StorageResult<()> {
```

Comment:
I think I'd prefer the name
```suggestion
    fn rollback_block_to(&self, height: u64) -> StorageResult<()> {
```